src/Symfony/Bundle/DependencyInjection/Compiler/SchemaFactoryPass.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Symfony\Bundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Inject in-use JSON Schema format into the factory.
+ *
+ * @internal
+ */
+final class SchemaFactoryPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container): void
+    {
+        if ($container->hasDefinition('api_platform.json_schema.schema_factory')) {
+            $container->getDefinition('api_platform.json_schema.schema_factory')->setArgument(6, '%api_platform.jsonschema_formats%');
+        }
+    }
+}
```

Comment:
we don't need a compiler pass for that right?

Reply:
Removed